lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -117,6 +118,7 @@ class OutlineTabSerializer(DatesBannerSerializerMixin, VerifiedModeSerializerMix
     enroll_alert = EnrollAlertSerializer()
     handouts_html = serializers.CharField()
     has_ended = serializers.BooleanField()
+    offer = serializers.DictField()
     offer_html = serializers.CharField()
```

Comment:
This PR leaves `offer_html` and `course_expired_html` in place. Once the MFE is upgraded to consume the new data versions, I can add a PR that will drop these old versions.